### PR TITLE
Test to validate available size in object bucket. CustoBZ(1880747 and 1880748)

### PR DESCRIPTION
**Pull request by parikshithb**
Covers bz: 1880747 and 1880748

Signed-off-by: Parikshith <pbyregow@redhat.com>

### CODE REVIEW

**Review by shreekarSS — DISMISSED** [review]

**Review by Shrivaibavi — APPROVED** [review]

File: tests/manage/rgw/test_object_bucket_size.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import logging
+
```
Comment (PrasadDesala) [inline tests/manage/rgw/test_object_bucket_size.py:2]: Delete the empty line

File: tests/manage/rgw/test_object_bucket_size.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import pytest
+
+from ocs_ci.framework.testlib import (
+    tier1,
+    config,
+    bugzilla,
+    skipif_ocs_version,
+)
+from ocs_ci.ocs import constants
+
```
Comment (PrasadDesala) [inline tests/manage/rgw/test_object_bucket_size.py:12]: Delete empty line

**Review by PrasadDesala — COMMENTED** [review]
LGTM, minor comments.

File: tests/manage/rgw/test_object_bucket_size.py (hunk 1 above)
Comment (parikshithb) [reply tests/manage/rgw/test_object_bucket_size.py]: Done

File: tests/manage/rgw/test_object_bucket_size.py (hunk 2 above)
Comment (parikshithb) [reply tests/manage/rgw/test_object_bucket_size.py]: Done

**Review by PrasadDesala — APPROVED** [review]

**Review by Shrivaibavi — APPROVED** [review]
LGTM

**Pull request merged**